scs2-examples/src/test/java/us/ihmc/scs2/examples/urdf/SimpleCrossFourBarRobotTest.java

```diff
@@ -41,7 +41,7 @@ public void testLoadingURDF() throws JAXBException, IOException
       parserProperties.setAutoGenerateCollisionName(false);
       parserProperties.setAutoGenerateVisualName(false);
       parserProperties.setTransformToZUp(false);
-      URDFModel originalURDFModel = URDFTools.loadURDFModel(inputStream, Collections.emptyList(), null, parserProperties);
+      URDFModel originalURDFModel = URDFTools.loadURDFModel(Collections.singletonList(inputStream), Collections.emptyList(), null, parserProperties);
```

Comment:
Shouldn't be needing this change right?

Reply:
Because the URDFParserProperties is set, we don't have a constructor that allows for this without doing a collection. We can either add another constructor, or have the singleton list. What do you think is better? (It may be easier on the user to add another constructor that takes in just one inputStream and all the other parameters as well)